lib/backend/model/ippool.go

```diff
@@ -93,6 +93,7 @@ func (options IPPoolListOptions) KeyFromDefaultPath(path string) Key {
 type IPPool struct {
 	CIDR          net.IPNet `json:"cidr"`
 	IPIPInterface string    `json:"ipip"`
+	IPIPOffSubnet bool      `json:"ipip_off_subnet"`
```

Comment:
As regard a mode field... yes that was another option and my original choice ... but may be tricky if there are various IPIP options that can be used together - in that case a mode field isn't that useful.  Also the default mode of "blank" isn't very intuitive.  Whereas a default of  false for "cross subnet only" is more explicit IMHO.

Anyways.... as I said above, I didn't mean this to still be in here, so for now I'm going to remove.